linera-client/src/error.rs

```diff
@@ -26,6 +26,14 @@ pub(crate) enum Inner {
     LocalNode(#[from] linera_core::local_node::LocalNodeError),
     #[error("remote node operation failed: {0}")]
     RemoteNode(#[from] linera_core::node::NodeError),
+    #[error("chain info response missing latest committee")]
+    ChainInfoResponseMissingCommittee,
+    #[error("arithmetic error: {0}")]
+    Arithmetic(#[from] linera_base::data_types::ArithmeticError),
+    #[error("invalid open message")]
+    InvalidOpenMessage(Option<linera_execution::Message>),
+    #[error("incorrect chain ownership")]
+    ChainOwnership,
```

Comment:
Most of these are the remote node's fault. I wonder whether we should extract some of that code and move it to `remote_node.rs`, and move the error variants there, too.

Reply:
https://github.com/linera-io/linera-protocol/issues/3445